Scripts/script-CommonServer.yml

```diff
@@ -8,6 +8,27 @@ script: |-
   // This script will be appended to each server script before being executed.
   // Place here all common functions you'd like to share between server scripts.
 
+  /**
```

Comment:
@anara123 better to do it same as String.prototype.format and with the documentation format. Otherwise it won't show in the script helper

Reply:
fixed

Reply:
@anara123 is it possible to also remove the "if"?(same as in string format)
The auto-doc doesn't recognize it as a function and so it's not being added to the doc.

Reply:
it is better not be removed, this how polyfill should be written. But it can be done